### PR TITLE
[DA-232] Problem in lookup gavs for patern in version

### PR DESCRIPTION
**Pull request by jomrazek**

### CODE REVIEW

**Comment by jomrazek** [comment]
retest this please

**Comment by jomrazek** [comment]
retest this please

**Comment by jbartece** [comment]
retest this please

File: common/src/main/java/org/jboss/da/common/version/VersionParser.java [hunk 1]
```diff
@@ -48,7 +48,8 @@ public String removeRedhatSuffix(String version) {
 
     public Matcher getVersionMatcher(String version) {
         String nonRedhatVersion = removeRedhatSuffix(version);
-        return Pattern.compile(nonRedhatVersion + PATTERN_SUFFIX_BUILT_VERSION).matcher("");
+        return Pattern.compile(Pattern.quote(nonRedhatVersion) + PATTERN_SUFFIX_BUILT_VERSION)
+                .matcher("");
```
Comment (jbartece) [inline common/src/main/java/org/jboss/da/common/version/VersionParser.java]: What is the expected behavior here?

Comment (jomrazek) [inline common/src/main/java/org/jboss/da/common/version/VersionParser.java]: @jbartece wraps string in regex quotes so we're safe from unsafe characters, as far as I understand

**Comment by jomrazek** [comment]
retest this please

**Comment by jbartece** [comment]
:+1:

**Comment by jomrazek** [comment]
Tested locally

**Pull request merged**